app/config.js

```diff
@@ -8,7 +8,9 @@ const notify = require('./notify');
 const Config = require('electron-config');
 
 // local storage
-const cache = new Config();
+const winCfg = new Config({name: 'winConf'});
+winCfg.defaultPosition = [50, 50];
+winCfg.defaultSize = [540, 380];
```

Comment:
Ya, trying to set it like 
`const winCfg = new Config({'defaultPosition':[50, 50], 'defaultSize': [540, 380]});`

But I don't think its the right way.